optimism/client.go

```diff
@@ -117,6 +115,10 @@ type Client struct {
 	traceSemaphore  *semaphore.Weighted
 	filterTokens    bool
 	supportedTokens map[string]bool
+	supportsSyncing bool
+	skipAdminCalls  bool
+	supportsPeering bool
+	bedrockBlock    *big.Int
```

Comment:
shouldn't there be another `eth.TraceConfig` for op-geth's built-in callTracer? The legacy call_tracer.js tracer isn't compatible with op-geth.